From stable-owner@vger.kernel.org Thu Jan  5 08:17:13 2023
From: Eric Biggers <ebiggers@kernel.org>
Date: Wed,  4 Jan 2023 23:13:50 -0800
Subject: ext4: remove unused enum EXT4_FC_COMMIT_FAILED
To: stable@vger.kernel.org
Cc: linux-ext4@vger.kernel.org, Ritesh Harjani <riteshh@linux.ibm.com>, Jan Kara <jack@suse.cz>, Harshad Shirwadkar <harshadshirwadkar@gmail.com>, Theodore Ts'o <tytso@mit.edu>
Message-ID: <20230105071359.257952-2-ebiggers@kernel.org>

From: Eric Biggers <ebiggers@kernel.org>

From: Ritesh Harjani <riteshh@linux.ibm.com>

commit c864ccd182d6ff2730a0f5b636c6b7c48f6f4f7f upstream.

Below commit removed all references of EXT4_FC_COMMIT_FAILED.
commit 0915e464cb274 ("ext4: simplify updating of fast commit stats")

Just remove it since it is not used anymore.

Signed-off-by: Ritesh Harjani <riteshh@linux.ibm.com>
Reviewed-by: Jan Kara <jack@suse.cz>
Reviewed-by: Harshad Shirwadkar <harshadshirwadkar@gmail.com>
Link: https://lore.kernel.org/r/c941357e476be07a1138c7319ca5faab7fb80fc6.1647057583.git.riteshh@linux.ibm.com
Signed-off-by: Theodore Ts'o <tytso@mit.edu>
Signed-off-by: Eric Biggers <ebiggers@google.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 fs/ext4/fast_commit.h |    1 -
 1 file changed, 1 deletion(-)

--- a/fs/ext4/fast_commit.h
+++ b/fs/ext4/fast_commit.h
@@ -93,7 +93,6 @@ enum {
 	EXT4_FC_REASON_RENAME_DIR,
 	EXT4_FC_REASON_FALLOC_RANGE,
 	EXT4_FC_REASON_INODE_JOURNAL_DATA,
-	EXT4_FC_COMMIT_FAILED,
 	EXT4_FC_REASON_MAX
 };
 
